Dockerfile

```diff
@@ -102,11 +104,14 @@ RUN go get -u github.com/mwitkow/go-proto-validators/protoc-gen-govalidators
 # Add Ruby Sorbet types support (rbi)
 RUN go get -u github.com/coinbase/protoc-gen-rbi
 
-RUN go get github.com/gomatic/renderizer/cmd/renderizer
+RUN go get github.com/gomatic/renderizer/v2/cmd/renderizer
 
 # Origin protoc-gen-go should be installed last, for not been overwritten by any other binaries(see #210)
 RUN go get -u github.com/golang/protobuf/protoc-gen-go
 
+# Need to get this too:
+RUN go get -u github.com/mwitkow/go-proto-validators
```

Comment:
This seems not have been updated in over a year. Is this safe to use?

Reply:
No idea! Not sure whether it's actually needed, in fact. Anyone here know?

Reply:
Is this instead of what is in line 102?
Shouldn't we specify the version that we need to get?

Reply:
"Is this safe to use?": Looks like _we_ don't use this at all; it is made available to (external) clients of this repo to optionally generate validators via one or two means. Besides not having been updated in over a year (which, by itself, is not necessarily a bad thing), it hasn't been upgraded, nor perhaps even tested, with any supported versions of Go that I can see.

So I think it neither hurts nor helps _us_ to include (this latest version) or, if we decide, remove it altogether (along with the `all/entrypoints.sh` option that runs it).

"Is this instead of what is in line 102?": Seems that line 102 is no longer necessary with the new line.

"Shouldn't we specify the version that we need to get?": Yes we should, thanks!

Reply:
Now comes the question of what to do about this stuff entirely:

- It's old and doesn't evidently support any supported version of Go
- It's apparently undocumented (unless one considers `all/entrypoint.sh` to be documentation!)
- It's unused internally (here at Namely)
- It's got no tests, e.g. to make sure that an `import` would actually work and/or the `--go-proto-validator` option to that script would not only work, but generate validator code that would actually build with Go 1.17

Do we know whether anyone is actually using it?

Personally I'd rather we just delete it than release it without some decent testing, but the latter would seemingly add substantial time (maybe a day or two?) to pushing this out the door.

Seems like we could (quietly?) drop support for this now and then, _if_ anyone asks, look into what it'd take to bring "modern" (Go-1.17) validation back on a more-leisurely schedule.

@idorekem : What do you think?

Reply:
Update: I just noticed that the `--with-validator` option, which specifies a _different_ validation method, is tested and passes with the updates I've made to the `Dockerfile` to address earlier concerns. (The `all/entrypoints.sh` script rejects specifying both `--with-validator` and `--go-proto-validator`.)

So, it seems (if my assumptions about what's going on here are correct) we would still be supporting validation in the general sense, just not this old way of doing things, if we simply removed it.